media-compose/src/androidMain/kotlin/dev/icerock/moko/media/compose/BindMediaPickerEffect.android.kt

```diff
@@ -21,8 +20,10 @@ actual fun BindMediaPickerEffect(mediaPickerController: MediaPickerController) {
     val context: Context = LocalContext.current
 
     LaunchedEffect(mediaPickerController, lifecycleOwner, context) {
-        val fragmentManager: FragmentManager = (context as FragmentActivity).supportFragmentManager
+        val activity: FragmentActivity = checkNotNull(context as? FragmentActivity) {
```

Comment:
can we use ComponentActivity here?

Reply:
not yet. for now we use `fragmentManager` for create and find `FilePickerFragment`, `ImagePickerFragment` and `MediaPickerFragment` and perhaps we can use ComponentActivity after change request result approach